specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-02-01/managedClusters.json

```diff
@@ -2349,6 +2356,7 @@
       "type": "string",
       "default": "Regular",
       "enum": [
+        "Spot",
         "Low",
```

Comment:
Do we still need to keep "low"? Is that still a valid option?

Reply:
Discussed in the meeting. We decided not to remove it for now and remove it in the next api version.